x-pack/plugins/actions/server/builtin_action_types/es_index.ts

```diff
@@ -38,6 +39,15 @@ export type ActionParamsType = TypeOf<typeof ParamsSchema>;
 //   eventually: https://github.com/elastic/kibana/projects/26#card-24087404
 const ParamsSchema = schema.object({
   documents: schema.arrayOf(schema.recordOf(schema.string(), schema.any())),
+  indexOverride: schema.nullable(
+    schema.string({
+      validate: (pattern) => {
+        if (!pattern.startsWith('alert-history-')) {
+          return `index must start with "alert-history-"`;
```

Comment:
**nit:** should this be `ALERT_HISTORY_PREFIX`?

```suggestion
        if (!pattern.startsWith(ALERT_HISTORY_PREFIX)) {
          return `index must start with "${ALERT_HISTORY_PREFIX}"`;
```

Reply:
Updated in https://github.com/elastic/kibana/pull/94909/commits/5aedddb16b8ab2fe9d1f090a1aef7eea0c7c36d0